src/main/kotlin/no/nav/familie/ba/sak/behandling/grunnlag/personopplysninger/PersongrunnlagService.kt

```diff
@@ -139,6 +140,7 @@ class PersongrunnlagService(
                         personinfoManuell.statsborgerskap?.map { GrStatsborgerskap.fraStatsborgerskap(it, person) } ?: emptyList()
                 person.bostedsadresser =
                         personinfoManuell.bostedsadresser.map { GrBostedsadresse.fraBostedsadresse(it, person) }.toMutableList()
+                person.sivilstandHistorisk = personinfo.sivilstandHistorikk.map { GrSivilstand.fraSivilstand(it, person) }
```

Comment:
Her kan vi vel bruke personinfoManuell som gjør historisk request? :) 
```suggestion
                person.sivilstandHistorisk = personinfoManuell.sivilstandHistorikk.map { GrSivilstand.fraSivilstand(it, person) }
```